GVFS/GVFS.Common/Maintenance/PrefetchStep.cs

```diff
@@ -174,7 +176,6 @@ private bool TryGetMaxGoodPrefetchTimestamp(out long maxGoodTimestamp, out strin
                     {
                         firstBadPack = i;
 
-                        metadata.Add("Errors", indexResult.Errors);

```

Comment:
Why was this line removed?

Reply:
To keep things simple I've updated all processes to rely on RunGitCommand to print errors.  In there we truncate based on size.

For this specific error, it won't be large since we make it ourselves.  However it would be a duplicate error in telemetry and I'd like the behavior to be consistent.  A more intrusive change that could be considered (for the future) would be to only return an exitCode from RunGitCommand.

Reply:
Ok thanks!